pacai/bin/capture.py

```diff
@@ -832,7 +832,8 @@ def main(argv):
 
     argv already has the executable stripped.
     """
-
+    logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
```

Comment:
config issue (this is one of the files you want to add it in, but you want to allow it to change here depending on the input from the user).